### DISCUSSION TITLE
Best way to draw virtual ink with different pressure?

### DISCUSSION THREAD

**Discussion by ctwhome** [pixijs/pixijs#10372] [v8 General] [answered]
New to Pixi 😅 

What would be the best way to draw virtual ink like TLDRAW does here? [Tweet](https://twitter.com/tldraw/status/1735707775366238465)
![hey](https://github.com/pixijs/pixijs/assets/4195550/49939b8e-47ab-4155-8b81-46870fdac734)

I guess this is a mesh that is being updated on each pointer move. I tried to replicate it with GraphicsPath, but so far I don't know how to connect the points to make a mesh.

**Comment by talhaozdemir** [comment]
I would recommend [Perfect Freehand](https://github.com/steveruizok/perfect-freehand), I used this plugin with PixiJS for one of my projects.

**Reply by ctwhome** [reply, marked as answer]
that is exactly what I was looking thanks so much!
